src/main/kotlin/lotto/presentation/InputView.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.presentation
+
+object InputView {
+    fun inputPurchaseAmount(): String? {
+        println(GUIDE_INPUT_PURCHASE_AMOUNT)
+        val purchaseAmount = readlnOrNull()
+        return purchaseAmount
```

Comment:
이렇게하면 불필요한 변수 선언을 줄일 수 있을 것 같아요. 😃

```suggestion
        return readlnOrNull()
```

Reply:
네 말씀하신대로 수정하겠습니다.